Sources/NIOKit/NIOKitTestCase.swift

```diff
@@ -0,0 +1,35 @@
+import NIO
+import XCTest
```

Comment:
I'm really surprised this is working. I tried something like this in Vapor 2 and it did not work. Maybe SPM has improved? I think this could potentially cause problems if this package is used as a dependency only?

Reply:
I’ll test it with it being a dependency